plugins/broadcast/plugin.go

```diff
@@ -0,0 +1,85 @@
+package broadcast
+
+import (
+	"github.com/iotaledger/goshimmer/plugins/broadcast/server"
+	"github.com/iotaledger/goshimmer/plugins/config"
+	flag "github.com/spf13/pflag"
+	"sync"
+
+	"github.com/iotaledger/hive.go/daemon"
+	"github.com/iotaledger/hive.go/events"
+	"github.com/iotaledger/hive.go/logger"
+	"github.com/iotaledger/hive.go/node"
+
+	"github.com/iotaledger/goshimmer/packages/shutdown"
+	"github.com/iotaledger/goshimmer/packages/tangle"
+	"github.com/iotaledger/goshimmer/plugins/messagelayer"
+)
+
+const (
+	pluginName = "Broadcast"
+	bindAddress = "broadcast.bindAddress"
+)
+
+var (
+	// plugin is the plugin instance of the activity plugin.
+	plugin *node.Plugin
+	once   sync.Once
+
+	log *logger.Logger
+)
+
+// Plugin gets the plugin instance.
+func Plugin() *node.Plugin {
+	once.Do(func() {
+		plugin = node.NewPlugin(pluginName, node.Enabled, configure, run)
+	})
+	return plugin
+}
+
+// Handler functions
+func init() {
+	flag.String(bindAddress, ":5050", "the bind address for the broadcast plugin")
+}
+
+// Configure events
+func configure(_ *node.Plugin) {
+	plugin.LogInfof("starting node with broadcast plugin")
+	log = logger.NewLogger(pluginName)
```

Comment:
Please use the integrated logger everywhere instead of creating a `log`.

Reply:
I am already using the logger from "github.com/iotaledger/hive.go/logger" as just like in other plugins

Reply:
Ok, correction: using the new logger now.
Not many plugins have that implemented that yet